### PR TITLE
Update issue templates for openjdk-build

### PR DESCRIPTION
**Pull request by sxa**
Make the templates a bit more useful including default labels which can be used for build break issues, including a list of triage questions. I have also reduced the number of "other" things like installer/api/website options to reduce complexity since it is now reasonably well-known that we have other repositories so this isn't the "dumping ground" it used to be :-)

Signed-off-by: Stewart X Addison <sxa@redhat.com>

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+---
+name: 😓 Build break
+about: Use this to report a broken build in jenkins
```
Comment (karianna) [inline .github/ISSUE_TEMPLATE/bug.md:3]: Is this a default?

File: .github/ISSUE_TEMPLATE/not-sure-where-to-raise-an-issue.md [hunk 2]
```diff
@@ -4,4 +4,10 @@ about: No sure where to raise the issue?
 
 ---
 
-Take a look at our TSC repository directory [here](https://github.com/AdoptOpenJDK/TSC/#dependent-projects). If you're still not sure then join our [slack channel[(http://adoptopenjdk.net/slack) and ask on `#general`
+Unsure where to raise something? Here' are some of the common places:
```
Comment (karianna) [inline .github/ISSUE_TEMPLATE/not-sure-where-to-raise-an-issue.md:7]: Typo

**Review by karianna — CHANGES_REQUESTED** [review]

File: .github/ISSUE_TEMPLATE/bug.md (hunk 1 above)
Comment (sxa) [reply .github/ISSUE_TEMPLATE/bug.md:3]: No - why do you ask?

File: .github/ISSUE_TEMPLATE/not-sure-where-to-raise-an-issue.md (hunk 2 above)
Comment (sxa) [reply .github/ISSUE_TEMPLATE/not-sure-where-to-raise-an-issue.md:7]: Fixed

File: .github/ISSUE_TEMPLATE/bug.md (hunk 1 above)
Comment (sxa) [reply .github/ISSUE_TEMPLATE/bug.md]: Ah ... because it's a duplicate of buildbreak.md ... Removed

**Review by karianna — APPROVED** [review]

**Review by gdams — APPROVED** [review]

**Pull request merged**